### ISSUE TITLE
Multiple value support for JQL `commitProperty`

### ISSUE THREAD

**Issue by antch** [javers/javers#912]
Currently there is no way to query on a commit property with multiple values using "IN" semantics, instead you have to query multiple times which is very inefficient:

```
Sets.newHashSet("1b2e", "8f32").stream()
    .flatMap(v -> javers.findSnapshots(QueryBuilder
        .byClass(myClass)
        .withCommitProperty("traceId", v)
        .build()).stream());
```

It would be much nicer to be able to simply query by multiple property values, for example:

```
javers.findSnapshots(QueryBuilder
    .byClass(myClass)
    .withCommitProperties("traceId", Sets.newHashSet("1b2e", "8f32"))
    .build());
```

**Comment by bartoszwalacik** [comment]
Hi @antch , now we have AND semantic for commit props, and you need OR, right?
First, let's talk about API changes in `QueryBuilder`, what do you suggest?

**Comment by antch** [comment]
This seems like a straight-forward enhancement so I could probably contribute, but unfortunately it has to be outside of work time.

**Comment by bartoszwalacik** [comment]
We all here have out day jobs ...

OR/AND semantic should be expressed in methods names

`.withAllCommitProperties()`

`.withAnyCommitProperty()`

**Comment by antch** [comment]
Good point. What about `withCommitPropertyIn(String, Collection<String>)` ?

My comment about work was only to communicate that I am not allowed to contribute to OSS during work time even if used AT work. I will be happy to do the enhancement as time permits.

**Comment by bartoszwalacik** [comment]
najs,
`withCommitPropertyIn(String, Collection<String>)` seems legit. What happens if I call it multiple times?

Please add new test case in `JaversRepositoryE2ETest`

**Comment by antch** [comment]
I'd say that the last invocation wins (overwrites), to be consistent with the other builder methods.

**Comment by bartoszwalacik** [comment]
The whole builder works in this way, if you call a method, it appends another predicate to the Query, for example:

```java 
    /**
     * Only snapshots with a given commit property.
     * <br/><br/
     *
     * If this method is called multiple times,
     * <b>all</b> given properties must match with persisted commit properties.
     * @since 2.0
     */
    public QueryBuilder withCommitProperty(String name, String value) 
```

So multiple method calls are joined with logical AND.

**Comment by antch** [comment]
If you call with a different property name that is true. But, if you call with the same property name multiple times, the last invocation wins. `withCommithPropertyIn` should work the same way.

**Comment by hungbang** [comment]
Hi @all
I have a similar question here: #919 . Thank you in advance

**Comment by lucas-prestes** [comment]
is anyone still working in this? If not I think I can contribute

**Comment by bartoszwalacik** [comment]
Nobody is working on this, please go on

**Comment by bartoszwalacik** [comment]
@lucas-prestes thanks for this contribution. I will release it tomorrow. 

btw. your commits are not linked to your github account, it happens when you commit with an email not registered here https://github.com/settings/emails

If you link them, you will be automatically listed on https://javers.org as a contributor

**Comment by bartoszwalacik** [comment]
released in 6.4.1
